book/website/project-design/data-governance/data-gov-ml.md

```diff
@@ -0,0 +1,67 @@
+- Data Collection
+- Data Management
+- Data Processing
+
+## Data Collection
+
+Many ML models are trained using datasets collected by the research team, which may be proprietary, or by using an open dataset that is available for download.
+The deep learning (DL) family of models, in particular, rely on massive corpuses of data such as text, code, images, sound, and other media.
+The process of data collection depends on the type and volume of data required and sources for acquisition.
+Many DL models rely on Internet data due to the sheer volume of digital content that is available on the web. 
+For example, [ImageNet datasets](https://www.image-net.org/about.php) are sourced from web images from image hosting websites like Flickr, and [LAION datasets](https://laion.ai/faq/) come from web crawling sources like Common Crawl.
+These methods of data collection through web scraping have raised issues regarding data quality and bias due to the nature of uncurated Internet data, as well as issues of data rights as web scraped data sets often include copyrighted data or data obtained without consent that may be in violation of the data license and terms of use.
+To address the issues of respecting data licenses, some ML teams are choosing to only use open source or permissively licensed data.
+This can be challenging to accomplish with data that does not have explicit licenses, or for data scraped for the web which may not have a license included in the metadata.
+
+### Governance Tool: Datasheets
+
+*We use the [Datasheets for Datasets](https://www.microsoft.com/en-us/research/uploads/prod/2019/01/1803.09010.pdf) from Microsoft in this chapter*.
+
+Datasheets are resources accompanying datasets that can provide key information about the motivation to create the dataset, 
+the data collection process, the dataset composition and pre-processing, and legal and ethical considerations when using the data.
+These resources helps contextualise the resource and is an opportunity for dataset creators to share any concerns, suggestions,
+and other advice regarding responsible use of the dataset.
+The Datasheets for Datasets paper contains a list of example questions that data creators should ask themselves and document.
+
+## Data Management
+
+Once data collection is complete, challenges emerge around data storage, access, and making changes.
+Because state-of-the-art ML datasets are very large, they tend to be stored in a central location, often a cloud provider database, and made available for download under certain circumstances.
+In addition to creating a governance structure for the opening or gating access to certain users, there is the challenge of creating a governance structure to allow people to request to opt-out their data from the dataset.
+Because opt-out occurs after a dataset has already been created, it will not remove the data from original versions of the dataset, but can facilitate removal from future versions.
```

Comment:
Is this separate from granting rights before the fact? I.e. people who agree to have their data harvested should not be allowed to opt-out in that sense. Additionally, I thought all the data was to be onlywith permission / permissive licensing, so this seems like an oversight / complaint mechanism?

Reply:
This is separate because after data has been collected, opt-in no longer an option.

Some teams think that data creators should have the option to opt-out even if their data is permissively licensed. This might be because the data creators did not give permission for their data to be used for training an LLM, for example.